.github/workflows/pages-source-setup.yml

```
@@ -0,0 +1,30 @@
+name: Configure GitHub Pages Source
+
+# One-time (or occasional) setup: switch Pages publishing source to GitHub Actions.
+# Run manually from the Actions tab after forking or if Pages is set to "Deploy from branch".
+on:
+  workflow_dispatch:
+
+permissions:
+  administration: write
```

actionlint (1.7.12)

[error] unknown permission scope "administration". all available permission scopes are "actions", "artifact-metadata", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)